debian/changelog

```diff
@@ -1,3 +1,16 @@
+imx-gpu-viv (6.4.11.p2.8-1) bookworm; urgency=medium
+
+  * Team upload.
+  * New upstream release 6.4.11.p2.8.
+  * Remove imx-gpu-viv-tools package which was dropped by NXP.
+      - d/control: drop imx-gpu-viv-tools stanza.
+      - d/imx-gpu-viv-tools.install: remove file.
+  * d/imx-gpu-wayland.symbols: update symbols from new upstream release.
+  * d/rules: bump imx-gpu-viv version to 6.4.11.p2.8.
+  * workflows: deb-builder: fix package name and add our repo to feeds.
+
+ -- Carlos Henrique Lima Melara <charlesmelara@riseup.net>  Fri, 27 Dec 2024 10:47:51 -0300
```

Comment:
Hey @charles2910, can you `export DEBEMAIL=...@toradex` and then run `debchange`?

We probably need some automated tooling to take care of these checks for us.

Reply:
Oh boy... I developed in the macbook and apparently I don't have the right export in there. Will fix it!

Reply:
No worries, it's probably my fault most of all because we really should have some checks developed for this months ago.